README.md

```diff
@@ -314,7 +313,10 @@ the mocha `done` function as the callback:
 var nemo;
 describe('my nemo suite', function() {
   before(function(done) {
-    nemo = Nemo(config, done);
+    Nemo(config, function(err, resolvedNemo) {
```

Comment:
From a consistent style point of view, you should have a space between `function` and `(` to keep the same style the rest of the file uses.